tests/expr_and_series/nth_test.py

```diff
@@ -18,21 +19,43 @@
         ([0, 2], {"a": [1, 3, 2], "z": [7.1, 8, 9]}),
     ],
 )
-def test_nth(
-    constructor: Constructor, idx: int | list[int], expected: dict[str, list[int]]
+def test_nth_not_polars(
+    not_polars_constructor: Constructor,
+    idx: int | list[int],
+    expected: dict[str, list[int]],
 ) -> None:
-    df = nw.from_native(constructor(data))
-    pl = get_polars()
-    is_polars = isinstance(df.to_native(), (pl.DataFrame, pl.LazyFrame))
-    if pl.build_info().get("version") is not None:
-        pl_version = tuple(int(i) for i in pl.build_info()["version"].split("."))
-    else:
-        pl_version = (0, 0, 0)
-    if is_polars and pl_version < (0, 20, 26):
-        with pytest.raises(
-            AttributeError, match="`nth` is only supported for Polars>=0.20.26."
-        ):
-            df.select(nw.nth(idx))
-    else:
-        result = df.select(nw.nth(idx))
-        compare_dicts(result, expected)
```

Comment:
Hey @Cheukting , I would suggest going another direction instead of adding additional fixtures.

You can:

```py
def test_nth_not_polars(
    constructor: Constructor,
    idx: int | list[int],
    expected: dict[str, list[int]],
    request: Any
) -> None:
    pl = get_polars()
    if "polars" in str(constructor) and parse_version(pl) < parse_version(0.20.26):
        request.applymarker(pytest.mark.xfail)
    
    ...
```